src/browser/dlc/utils/ContractUpdater.ts

```diff
@@ -800,20 +800,14 @@ export class ContractUpdater {
     }
   }
 
-  isOfferedOrAccepted(
-    contract: AnyContract
-  ): contract is OfferedContract | AcceptedContract {
-    return (
-      contract.state === ContractState.Accepted ||
-      contract.state === ContractState.Offered
-    )
-  }
-
   async toFailedContract(
     contract: InitialContract | OfferedContract | AcceptedContract,
     reason: string
   ): Promise<FailedContract> {
-    if (this.isOfferedOrAccepted(contract)) await this.unlockUtxos(contract)
+    if (
+      isContractOfState(contract, ContractState.Offered, ContractState.Accepted)
```

Comment:
Can you extract this in a variable before the `if`? the indent is really weird now.

Reply:
well, I cannot extract the function itself because a typeguard can only be used in an `if` statement but I extracted the states list into a statically `const`, the indent seems better now, if you can confirm :)

Reply:
Ah yeah makes sense. But looks good thanks!